doc/pins/p4rt_app_hld.md

```diff
@@ -0,0 +1,169 @@
+# P4RT Application HLD
+
+_Rev v0.1_
+
+- [Revision](#revision)
+- [Scope](#scope)
+- [Definitions/Abbreviations](#definitions-abbreviations)
+- [Overview](#overview)
+- [Requirements](#requirements)
+- [P4RT Application High-Level Design](#p4rt-application-high-level-design)
+  * [Arbitration & Controller Roles](#arbitration---controller-roles)
+  * [P4 programs & P4Info](#p4-programs---p4info)
+  * [ACL Table Definitions](#acl-table-definitions)
+  * [Writing (insert/modify/delete) into the P4RT APPL_DB Table](#writing--insert-modify-delete--into-the-p4rt-appl-db-table)
+  * [WCMP/Hashing](#wcmp-hashing)
+  * [Response path](#response-path)
+- [APPL DB Schema High-Level Design](#appl-db-schema-high-level-design)
+- [Testing Requirements/Design](#testing-requirements-design)
+- [Open/Action items - if any](#open-action-items---if-any)
+
+## Revision
+
+Rev  | RevDate    | Author(s)   | Change Description
+---- | ---------- | ----------- | ------------------
+v0.1 | 06/30/2021 | Google, ONF | Initial Version
+
+## Scope
+
+This document describes the high-level design for adding a P4Runtime gRPC service into SONiC, and outlines the Application DB changes needed.
```

Comment:
Since we already have sonic-telemetry to handle gRPC requests, it would be better to mention the unique port that we're planning to use in the gRPC requests for communication between the controller and P4RT.

Reply:
Sure, we use port number 9559, will update the HLD.